.github/workflows/reusable-dev-deployment.yml

```diff
@@ -0,0 +1,160 @@
+name: '[reusable only] Development deployment'
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        required: true
+        type: string
+      sha:
+        required: false
+        type: string
+        default: ${{ github.sha }}
+      env:
+        required: true
+        type: string
+      pr_number:
+        required: false
+        type: string
+      dropDBOnUninstall:
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  dev-deployment:
+    name: Deploy to Kubernetes
+    runs-on: ubuntu-latest
+    environment:
+      name: ${{ inputs.env }}
+    steps:
+      - name: Get link to currently running job logs
+        uses: Tiryoh/gha-jobid-action@v0.1.2
+        id: job-url
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          job_name: Upgrade or install deployment (${{ inputs.name }}, ${{ inputs.sha }}, ${{ inputs.pr_number }}) / Deploy to Kubernetes
```

Comment:
Does this still work? Is the job title really this? Are you aware that GitHub truncates the job name, making it a little tricky?